app/models/order.rb

```diff
@@ -89,6 +89,25 @@ def set_initial_state
     self.state ||= :draft
   end
 
+  def is_priority?
```

Comment:
Just suggestion: This method name can be changed to `priority?` instead of `is_priority?`

Reply:
I kinda like is_priority bettrer 😢

Reply:
ohh Okay, Then keep it. No worries.